site/en/docs/privacy-sandbox/shared-storage/index.md

```diff
@@ -6,116 +6,149 @@ subhead: >
 
-The proposed Shared Storage API will allow sites to store and access
-unpartitioned cross-site data. This data must be read in a secure environment
-to prevent leakage. This API will work in combination with other proposals,
-such as [Trust Tokens](/docs/privacy-sandbox/trust-tokens/),
-[Fenced Frames](/docs/privacy-sandbox/fenced-frame/), and others.
+The Shared Storage API allows sites to store and access unpartitioned cross-site data. This data must be read in a secure environment to prevent leakage. 
 
 ### Who is this for?
 
-Many different organizations may benefit from using the Shared Storage API. For
-example:
-
-*  Content producers and adtechs to solve many common use cases, which currently
-   rely on third-party cookies.
-*  Payments providers to understand if the user is an existing customer and
-   tailor the checkout experience.
-*  Web security companies who use custom on-device logic to flag suspicious or
-   dangerous behavior.
-
-## Use cases for Shared Storage
-
-The Shared Storage API intends to support many use cases, replacing several
-existing uses for third-party cookies. This may include:
-
-*  [Frequency control](/docs/privacy-sandbox/shared-storage/frequency-control/)
-*  [A/B or lift experiments](/docs/privacy-sandbox/shared-storage/ab-testing/)
-*  [Creative rotation](/docs/privacy-sandbox/shared-storage/creative-rotation/)
-*  [Determining known customers](/docs/privacy-sandbox/shared-storage/known-customer/)
-*  Recording aggregated statistics, such as demographics, reach, frequency
-   measurement, and conversion measurement with the
-   [Private Aggregation API](/docs/privacy-sandbox/private-aggregation/)
-*  Confirm login for payment provider
-
-{% Aside %}
-While it is technically possible to configure and collect user consent status with Shared Storage, we don't recommend this use case.
+There are many different kinds of companies which may benefit from using the Shared Storage API. For example:
+
+*   Adtechs could measure campaign reach, set frequency caps, and rotate creatives, all of which currently rely on third-party cookies.
+*   Payments providers could determine if a user is an existing customer and tailor the checkout experience.
+*   Web security companies can build custom logic to flag suspicious or dangerous behavior.
+
+Is your company looking for cross-site storage solutions that haven’t yet been addressed? [Share your use case](https://github.com/WICG/shared-storage/issues).
+
+### Use cases
+
+The Shared Storage API intends to support many use cases, replacing several existing uses for third-party cookies. This may include:
+
+<table>
+  <tr>
+   <td style="background-color: #1967d2">Use Case 
+   </td>
+   <td style="background-color: #1967d2">Description
+   </td>
+   <td style="background-color: #1967d2">Shared Storage API Output Gates
+   </td>
+  </tr>
```

Comment:
```suggestion
<table class="with-heading-tint width-full">
  <thead>
  <tr>
   <th>Use case</th>
   <th>Description</th>
   <th>Output gate</th>
  </tr>
  </thead>
```

Reply:
@kevinkiklee the blue isn't accessible -- in the future, use these docs for creating tables with a tint: https://developer.chrome.com/docs/handbook/components/#tables-with-tinted-header